### DISCUSSION TITLE
Entangle directive cannot find the Livewire nested array property randomly

### DISCUSSION THREAD

**Discussion by phuclh** [livewire/livewire#5037] [7. Bugs (archived)]
The error happens randomly everytime I fresh the page. I found some answers on Livewire Discord but all of them told me to upgrade to Livewire v2.6.5 but I am using v2.10.5.

https://user-images.githubusercontent.com/6707194/173898082-210be41a-be06-4265-9a08-09d714063fe5.mp4

**Comment by joshhanley** [comment]
@phuclh can you share your component class and view? Thanks!
